crowbar_framework/config/repos-cloud.yml

```diff
@@ -97,3 +97,75 @@ suse-12.1:
       required: "recommended"
       url:
       ask_on_error: false
+opensuse-42.1:
+  x86_64:
+    leap-42_1:
+      name: "openSUSE-Leap"
+      required: "mandatory"
+      features: ["os", "ha", "ceph"]
+      repomd:
+        tag: ["obsproduct://build.opensuse.org/openSUSE:Leap:42.1/openSUSE/42.1/ftp/x86_64",
+              "obsproduct://build.opensuse.org/openSUSE:Leap:42.1/openSUSE/42.1/cd/x86_64"]
+        key_md5: "d8039df72c407bc7dd7c1d9c66d58627"
+      url: "http://download.opensuse.org/distribution/leap/42.1/repo/oss/"
+      smt_path:
+      ask_on_error: false
+    leap-42_1-update:
+      name: "openSUSE-Leap-Update"
+      required: "mandatory"
+      features: ["os", "ha", "ceph"]
+      repomd:
+        tag: "obsrepository://build.opensuse.org/openSUSE:Leap:42.1:Update/standard"
+        key_md5: "d8039df72c407bc7dd7c1d9c66d58627"
+      url: "http://download.opensuse.org/update/leap/42.1/oss/"
+      smt_path:
+      ask_on_error: false
+    chef-10:
```

Comment:
I explicitly didn't want these as I wanted the minimal set of packages for the nodes, to not impact anything else on Leap. I understand these are needed for the admin server, but the non-admin nodes don't need all the changed packages.